packages/functional-tests/pages/baseTokenCode.ts

```diff
@@ -23,7 +23,9 @@ export abstract class BaseTokenCodePage extends BaseLayout {
 
   get submitButton() {
     this.checkPath();
-    return this.page.getByRole('button', { name: 'Confirm' });
+    return this.page.getByRole('button', {
```

Comment:
Perhaps this should be two separate locators so we can accurately check for the right one? Not a deal breaker though, only if other changes are needed to pass CI